packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.test.tsx

```diff
@@ -1,51 +1,57 @@
 import React from 'react';
-import { mount, ReactWrapper } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import { ColorPickerPopover } from './ColorPickerPopover';
-import { ColorSwatch } from './ColorSwatch';
 import { createTheme } from '@grafana/data';
+import userEvent from '@testing-library/user-event';
 
 describe('ColorPickerPopover', () => {
   const theme = createTheme();
 
+  it('should be tabbable', () => {
+    render(<ColorPickerPopover color={'red'} onChange={() => {}} />);
+    const color = screen.getByRole('button', { name: 'red color' });
+    const customTab = screen.getByRole('button', { name: 'Custom' });
+
+    userEvent.tab();
+    expect(customTab).toHaveFocus();
+
+    userEvent.tab();
+    expect(color).toHaveFocus();
+  });
+
```

Comment:
Wohooo, great refactor from enzyme to rtl 🙌

Reply:
+1 on this, nice tests